src/components/Items/ProfileContact.js

```diff
@@ -61,7 +61,7 @@ const ProfileContact = ({
             <>
               {userId !== item.user_id && (
                 <View>
-                  <Mi name="arrow-forward-ios" size={18} />
+                  <Mi name="arrow-forward-ios" size={18} color={COLORS.white2} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `Mi` component has been changed from default to `COLORS.white2`. Ensure that this color change is consistent with the overall theme and visibility on all screens where this component is used.
```diff
- <Mi name="arrow-forward-ios" size={18} />
+ <Mi name="arrow-forward-ios" size={18} color={COLORS.white2} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->